### ISSUE TITLE
GitHub gists not displaying

### ISSUE THREAD

**Issue by loopDelicious** [postmanlabs/postman-blog#275]
Like the YT videos not displaying w/o embed, here is an example of gist not displaying:
https://blog.postman.com/api-testing-tips-from-a-postman-professional/

**Comment by tristandenyer** [comment]
I tried a few different WordPress supported shortcodes for Gists, but they do not seem to work in Preview. Will look into a way to support this after it comes across in GrahQL response.

**Comment by loopDelicious** [comment]
Here's another example of a gist not displaying: https://blog.postman.com/write-to-your-local-file-system-using-a-postman-collection/

**Comment by tristandenyer** [comment]
@loopDelicious Yeah, we haven't been able to implement a fix for this yet. All Gists are not displaying right now.

A workaround would be embedding them as code blocks, for now.
